examples/benchmarks/run_experiment_grid.py

```diff
@@ -22,23 +22,20 @@
 
 import omnisafe
 from omnisafe.common.experiment_grid import ExperimentGrid
-from omnisafe.typing import NamedTuple, Tuple
 
 
 def train(
     exp_id: str,
     algo: str,
     env_id: str,
-    custom_cfgs: NamedTuple,
-) -> Tuple[float, float, float]:
+    custom_cfgs: dict,
+) -> tuple:
```

Comment:
Should fill in the content type for generic containers. `dict` represents `dict[Any, Any]`.